src/__tests__/labs/ViewLabs.test.tsx

```diff
@@ -85,24 +84,20 @@ describe('View Labs', () => {
     it('should render a table with data', async () => {
       const { expectedLab } = setup([Permissions.ViewLabs, Permissions.RequestLab])
 
-      expect(screen.getByRole('columnheader', { name: /labs.lab.code/i })).toBeInTheDocument()
-      expect(screen.getByRole('columnheader', { name: /labs.lab.type/i })).toBeInTheDocument()
-      expect(
-        screen.getByRole('columnheader', { name: /labs.lab.requestedOn/i }),
-      ).toBeInTheDocument()
-
-      expect(
-        await screen.findByRole('columnheader', { name: /labs.lab.status/i }),
-      ).toBeInTheDocument()
+      const headers = await screen.findAllByRole('columnheader')
+      const cells = screen.getAllByRole('cell')
+
+      expect(headers[0]).toHaveTextContent(/labs\.lab\.code/i)
+      expect(headers[1]).toHaveTextContent(/labs\.lab\.type/i)
+      expect(headers[2]).toHaveTextContent(/labs\.lab\.requestedOn/i)
+      expect(headers[3]).toHaveTextContent(/labs\.lab\.status/i)
+      expect(cells[0]).toHaveTextContent(expectedLab.code)
+      expect(cells[1]).toHaveTextContent(expectedLab.type)
+      expect(cells[2]).toHaveTextContent(
```

Comment:
I'm not sure about this... This is now enforcing the order of of the columns - which may or may not be important :man_shrugging:

Reply:
> I'm not sure about this... This is now enforcing the order of the columns - which may or may not be important

If it's mock data then it is irrelevant, you would still just be asserting things are rendering. 😄